code/formatters.py

```diff
@@ -111,6 +111,7 @@ def every_word(word_func):
         first_vs_rest(lambda w: w.lower(), lambda w: "_" + w.lower()),
     ),
     "NO_SPACES": (NOSEP, every_word(lambda w: w)),
+    "ABBREVIATION": (NOSEP, every_word(lambda w: w[0])),
```

Comment:
As mentioned below, mixed feelings about this formatter.  If we do remove the use case, I'd also remove the formatter